packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -209,31 +185,58 @@ events.
+
+X&nbsp;events types can be triggered from different modules in the X&nbsp;Components project. However, every module has its own sort of components (e.g. Empathize X events, Search Box X events, etc.). See the corresponding `events.types.ts` file for each module in the [X&nbsp;Components library in GitHub](https://github.com/empathyco/x/tree/main/packages/x-components/src/x-modules).
+
+#### X API
 
-The X API allows your website to communicate with Interface X. It is a set of utilities that helps
-to integrate Interface X into your website.
+The [X&nbsp;API](https://github.com/empathyco/x/blob/main/packages/x-components/src/x-installer/api/base-api.ts) object allows your commerce store to communicate with Interface&nbsp;X. It supports multiple functions to integrate Interface&nbsp;X in your website.
 
 | Function         | Parameters                                                                     | Description                                                                                                                                                       |
 | ---------------- | ------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| init             | - [Snippet Configuration](#snippet-configuration) - The initialisation options | [Initialises Interface X on demand](#initialise-on-demand).                                                                                                       |
-| search           | - query (Optional) - The query to open Interface&nbsp;X with                   | Opens Interface&nbsp;X and triggers a search with the given query.                                                                                                |
-| setSnippetConfig | - [Snippet Configuration](#snippet-configuration) - The Initialisation options | Changes the options and all components react to the changes. Useful, for example, to change both search engine and displayed language without reloading the page. |
+| `init`             | [snippet configuration params](#snippet-configuration) - Initialization options | [Initializes Interface&nbsp;X on demand](#initializing-interface-x-project-on-demand).   |
+| `search`           | `query` - Query to open Interface&nbsp;X  | _Optional_. Executes Interface&nbsp;X and triggers a search with the given query.  |
+| `setSnippetConfig` | [snippet configuration params](#snippet-configuration) - Initialization options | Changes initialization options so that all components react to the changes, i.e. changing both search engine and language without reloading the page. |
```

Comment:
```suggestion
| `setSnippetConfig` | [snippet configuration params](#snippet-configuration) - Initialization options | Changes initialization options so that all components react to the changes, i.e. changing both search engine and language without reloading the page. |
| `addProductToCart` | none | To be called from PDP (Product Detail Page) when the user clicks on the Add To Cart Button. Calling this function will send tracking of the AddToCart event to the [Empathy Tagging API](https://docs.empathy.co/develop-empathy-platform/capture-interaction-signals/tagging-api-guide.html) for the product which is currently in screen.  |
```

I am adding this other event because yesterday we realized we didn't document it, and this is basic to integrate customers. cc @javieri-empathy

Reply:
To follow the same structure and style before, I'll rewrite the description field: 

> Sends tracking of the `AddToCart`event to the Empathy Tagging microservice for the selected /displayed product.

Can't commit your suggestion directly inhere. If you agree with the proposed description, I'll add the new info in a new commit together with the rest of changes.

Reply:
To follow the same structure and style before, I'll rewrite the description field: 

> Sends tracking of the `AddToCart`event to the Empathy Tagging microservice for the selected /displayed product.

Can't commit your suggestion directly inhere. If you agree with the proposed description, I'll added the new info in a new commit together with the rest of changes.

Reply:
I would just add that this function might have a parameter: the product id. If it is not provided X Components will try to use the url as a identifier, but it might fail.